netdata/netdata
# Disk metrics (Windows Plugin)

##### Summary
This PR is bringing Disk metrics for Windows Plugin.

Some metrics were still not added, because apparently they need a different algorithm.

##### Test Plan

1. Compile this branch
2. Generate installer
3. Install and verify your dashboard.

Linux was also modified, so:

1. Compile this branch on Linux
2. Start netdata to confirm charts are ok.

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
